### PR TITLE
Check coding style in CI builds

### LINKED ISSUE #702: Require conformance to style standards
#### Issue Summary

In the `CONTRIBUTING.md` file there is a section on Style Guidelines & Naming Conventions. Please review and implement a check in Travis CI. If there are any style errors, either fix or create a rule to exclude the error.

Before writing any code, we suggest you review our `CONTRIBUTING.md` file and propose your solution for discussion here in the comments. Thank you!

#### Aceptance Criteria

* TravisCI checks that the code conforms to our Style Guidelines & Naming Conventions as defined in our `CONTRIBUTING.md` file
* tests pass locally, in Travis CI and using Docker

### PR DESCRIPTION
**Pull request by GeoffYoung**
This will add a code sniffer check to the travis-ci build before running phpunit tests.

It also corrects the PSRr2 discrepancies currently in the /lib directory, which were almost entirely trailing whitespace issues.

resolves #702

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=707) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=707) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-php?pullRequest=707) it.</sub>

File: lib/SendGrid.php [hunk 1]
```diff
@@ -13,6 +13,8 @@
  * @link      http://packagist.org/packages/sendgrid/sendgrid
  */
 
+namespace SendGrid;
```
Comment (GeoffYoung) [inline lib/SendGrid.php]: This was the only questionable change. PSR2 requires a namespace to be set, so I'm just using `/SendGrid` based on the `$namespace` variable set at the beginning of the class.

**Comment by peter279k** [comment]
The Travis CI build is broken. After checking the Travis CI build log, the `SendGrid` class is not found.
The related exception message is as follows:

```
PHP Fatal error:  Class 'SendGrid' not found 
```

Can you fix this error?

**Comment by thinkingserious** [comment]
Hello @GeoffYoung,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @GeoffYoung,<br /><br/>Thanks again for the PR!<br /><br />                                         We noticed that you have earned 3 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold.<br /><br />                                         Also, please be sure that you have officially registered <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid DX

**Comment by thinkingserious** [comment]
Hello @GeoffYoung,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 3 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Comment by childish-sambino** [comment]
Duplicate of https://github.com/sendgrid/sendgrid-php/pull/725

**Pull request closed without merging**